hackathon_contributions_new_data_format/Isensee_JCB2018/visualization/main.py

```diff
@@ -0,0 +1,187 @@
+            #     # dont have to go in expCondFile, all Cond are 0
+            # else:
+            #     ind_expCond = experimental_condition.conditionId == conditionIds
+            #     #do smth with information in this file
+            # get measurement values for each condId
+
+            # # create empty dataframe for means and SDs
+            # ms_c = pd.DataFrame(columns=['mean', 'sd'], index=conditionIds)
+            #
+            # # group measurement values for each conditionId
+            # for ID in conditionIds:
+            #     ind_meas = ((measurement_data['simulationConditionId'] == ID) &
+            #                 (measurement_data['datasetId'] == datasetId))
+            #     ms_c.at[ID, 'mean'] = np.mean(measurement_data[ind_meas].measurement)
+            #     ms_c.at[ID, 'sd'] = np.std(measurement_data[ind_meas].measurement)
+
+            # group measurement values for each conditionId
+            ms = getDataToBePlotted.getDataToBePlotted(visualization_specification, measurement_data, uni_conditionIds, i,
+                                                       clmn_name_unique)
+
+            # barplot
+            x_pos = range(len(visualization_specification[ind_plot].index.values))         # how many x-values (how many bars)
+            x_name = visualization_specification[ind_plot].legendEntry[i]
+
+            ax[axx, axy].bar(x_name, ms['mean'], yerr=ms['sd'])
+            ax[axx, axy].set_title(visualization_specification.plotName[i],fontsize=10)
+
+        elif indepVar == 'time':
+
+            # obtain unique observation times
+            uni_times = np.unique(measurement_data[ind_dataset].time)
+            clmn_name_unique = 'time'
+
+            # # create empty dataframe for means and SDs
+            # ms = pd.DataFrame(columns=['mean', 'sd'], index=uni_times)
+            #
+            # # group measurement values for each conditionId
+            # for var_time in uni_times:
+            #     ind_meas = ((measurement_data['time'] == var_time) &
+            #                 (measurement_data['datasetId']==datasetId))
+            #     ms.at[var_time, 'mean'] = np.mean(measurement_data[ind_meas].measurement)
+            #     ms.at[var_time, 'sd'] = np.std(measurement_data[ind_meas].measurement)
+
+            # group measurement values for each conditionId/unique time
+            ms = getDataToBePlotted.getDataToBePlotted(visualization_specification, measurement_data, uni_times, i,
+                                                       clmn_name_unique)
+
+            ax[axx, axy].errorbar(uni_times, ms['mean'], ms['sd'], linestyle='-', marker='.',
+                         color=cmap[min(7,i-plotInd[i_plotId])],
+                         label=visualization_specification[ind_plot].legendEntry[i]
+                         )
+            ax[axx, axy].legend()
+            ax[axx, axy].set_title(visualization_specification.plotName[i], fontsize=10)
```

Comment:
Try not setting fontsizes per plot, but use matplotlib rcParams https://matplotlib.org/tutorials/introductory/customizing.html#matplotlib-rcparams